HISTORY.rst

```diff
@@ -9,6 +9,10 @@ Changelog
 
 version 1.7.0-dev
 ---------------------------
++ Add ``--standerderror-bytes`` or ``--sb`` option to change the maximum 
```

Comment:
In linux wordt het vooral `stderr` en `stdout` genoemd. Dus dat zou consistent gebruikt moeten woorden. ``--stderr-bytes``